### PR TITLE
add contact me page

### PR DESCRIPTION
**Pull request by olaruandreea**

### CODE REVIEW

File: static/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    background-image: -o-linear-gradient(top, #dbdbdb, #cccccc);
+    background-image: linear-gradient(top, #dbdbdb, #cccccc);filter:progid:DXImageTransform.Microsoft.gradient(GradientType=0,startColorstr=#dbdbdb, endColorstr=#cccccc);
+  }
+
+
+.flash {
+    background-color: #FBB0B0;
+    padding: 10px;
+    width: 400px;
+  }
```
Comment (SeanHealy33) [inline static/css/contact.css]: `  ` ⚡

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 /venv/*
 blog.db
-run.pyc
\ No newline at end of file
+run.pyc
+form.pyc
```
Comment (SeanHealy33) [inline .gitignore]: rather than update this everytime you add a new file you should add wildcard ignores here 

`*.pyc`

**Comment by SeanHealy33** [comment]
You've committed `forms.pyc` remove that from the commit :)

File: templates/base.html [hunk 3]
```diff
@@ -1,11 +1,13 @@
 <!doctype html>
 <html>
 <head>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <title>{% block title %}Recepies Blog{% endblock %}</title>
+    {% block head %}
```
Comment (SeanHealy33) [inline templates/base.html:4]: What does this `block head` give you exactly?

File: templates/contactme.html [hunk 4]
```diff
@@ -1,4 +1,52 @@
-{% extends "base.html" %}
-{% block content %}
+{% extends "base.html" %} 
+
+<!--extend the functionality from the header of the parent component.-->
+{% block head %}
+    {{ super() }}
```
Comment (SeanHealy33) [inline templates/contactme.html:5]: why does this blockhead call super but base does not?

File: .gitignore (hunk 2 above)
Comment (olaruandreea) [reply .gitignore]: done

Comment (olaruandreea) [reply .gitignore]: done

File: templates/contactme.html (hunk 4 above)
Comment (olaruandreea) [reply templates/contactme.html:5]: using super() in the child class allows me to extend the functionality of the parent class "(base.html)".

File: templates/base.html (hunk 3 above)
Comment (olaruandreea) [reply templates/base.html:4]: Block head contains a link pointing to the bootstrap cdn . 
Just a file required by bootstrap !

**Pull request merged**